IAteIt/IAteIt/ViewModel/FeedMealStore.swift

```diff
@@ -44,18 +44,25 @@ final class FeedMealStore: ObservableObject {
             }
             
             self.mealList = fetchedMealList
-            
-            for meal in self.mealList {
-                FirebaseConnector.shared.fetchMealComments(mealId: meal.id!) { [weak self] comments in
-                    DispatchQueue.main.async {
-                        self?.commentList[meal.id!] = comments
-                    }
+            try await fetchCommentsForFeed(with: self.mealList)
+        }
+    }
+    
+    private func fetchCommentsForFeed(with meals: [Meal]) async throws {
+        try await withThrowingTaskGroup(of: (mealId: String, comments: [Comment]).self) { group in
+            for meal in meals {
+                group.addTask {
+                    let comments = try await FirebaseConnector.shared.fetchMealComments(mealId: meal.id!)
+                    return (meal.id!, comments)
```

Comment:
```suggestion
                    let fetchedComments = try await FirebaseConnector.shared.fetchMealComments(mealId: meal.id!)
                    return (meal.id!, fetchedComments)
```

@sunshiningsoo 동의합니다! 요렇게 수정하면 어떨까요?

Reply:
완전 나이스입니다👍